src/components/accessories/autocompleteField/styles.scss

```diff
@@ -8,9 +8,8 @@
       top: 2px;
     }
   }
-  .MuiSelect-select.MuiInputBase-input {
-    padding-top: 15px;
-    padding-bottom: 15px;
+  .MuiInputBase-input {    
+    height: 1.8rem;
```

Comment:
Do not use rem inside height. Actually, we're not gonna use static height inside css components. This could lead to some side effects.

Reply:
OK. But the height of the autocomplete component is not the same as others components.

Reply:
Ok, you'll need to investigate: it could be some extra-padding or something like this. Keep this solutions as fallback.